airootfs/usr/share/vx-img/flash-image.sh

```diff
@@ -55,6 +64,22 @@ if [[ $_surface == 0 && $_haskeys == 0 ]]; then
             exit
         fi
     else
+        if [[ $_surface == 0 ]]; then
+            _setup=$(mokutil --sb-state | grep -q "Setup Mode")
+
+            if [[ -z $setup ]]; then
+                echo "Device is not in Setup Mode."
+                echo "Please reboot into the BIOS and enter Setup Mode before continuing."
+                echo "Reboot now? [Y/n]:"
```

Comment:
should we reboot into BIOS?

This post:
https://superuser.com/questions/519718/linux-on-uefi-how-to-reboot-to-the-uefi-setup-screen-like-windows-8-can

indicates it can be possibly be done with

```
systemctl reboot --firmware-setup
```

though I haven't tested it.

Reply:
Yes, this is a good point. I need to test that behavior.